aviary/docs/theory_guide/propulsion.ipynb

```diff
@@ -24,7 +24,7 @@
    "outputs": [],
    "source": [
     "# Testing Cell\n",
-    "# ********NOTE******** should we be using EngineModels vs EngineModel consistently here?\n"
+    "# ****NOTE**** should we be using EngineModels vs EngineModel consistently here?\n"
```

Comment:
I guess to be technically correct it could be `EngineModel` and `EngineModel`s but that might look very ugly. No one is usually confused when someone refers to Python objects as plural (numpy arrays, strings, etc.) so I didn't worry much about when I wrote these sections